[JA] Bump up basics/mix-tasks to 1.0.2

Tsuyoshi84 opened the pull request:
Translated the mix-tasks page to follow the latest version 1.0.2.
Please review the pull request @elixirschool/japanese

This pull request includes the following changes.
- https://github.com/elixirschool/elixirschool/commit/633a3ebe5fb8495b31937fa7e3fa4198cf4bdf9b#diff-ce81ef91d95296ee6fc601cb0c462f42
- https://github.com/elixirschool/elixirschool/commit/575bf3bf036a8dd23687426c0b53a536aba8dd6d#diff-ce81ef91d95296ee6fc601cb0c462f42
- https://github.com/elixirschool/elixirschool/commit/8a8f948636d464505a47856cd05728504fe30d78#diff-ce81ef91d95296ee6fc601cb0c462f42


Thank you 😊

riseshia commented:
I think some changes not included in this pr. i.e. https://github.com/elixirschool/elixirschool/commit/633a3ebe5fb8495b31937fa7e3fa4198cf4bdf9b#diff-ce81ef91d95296ee6fc601cb0c462f42R64

could you check this? XD

Tsuyoshi84 commented:
Oh, seems like somehow I reverted one change that I should have made. 

I added the missing change at line 64. 
Thank you for your careful checking!

riseshia commented on ja/lessons/basics/mix-tasks.md:
````diff
@@ -111,3 +121,6 @@ mix app.start         # Starts all registered apps
 ...
 ```
+
+注: 新しいタスクが `mix help` の出力に表示されるには、私たちのコードが事前にコンパイルされてなければいけません。
````
In my opinion,`私たちの` is not not necessary in Japanese, How do you think?

riseshia commented on ja/lessons/basics/mix-tasks.md:
````diff
@@ -111,3 +121,6 @@ mix app.start         # Starts all registered apps
 ```
+
+注: 新しいタスクが `mix help` の出力に表示されるには、私たちのコードが事前にコンパイルされてなければいけません。
+私たちは `mix compile` を直接実行するか、先ほどと同じように `mix hello` を実行してコンパイルのトリガーを作り出すことでコンパイルすることができます。
````
Same here `私たちは`

riseshia replied:
(maybe) `コンパイルのトリガーを作り出すことでコンパイルすることができます。` -> 
`コンパイルできます。`
It seems too verbose.

Tsuyoshi84 commented:
I modified the translation.
Please check it out!
